libfwupdplugin/fu-context.h

```diff
@@ -128,6 +128,15 @@ typedef enum {
 	 * Since: 2.0.2
 	 **/
 	FU_CONTEXT_FLAG_INHIBIT_VOLUME_MOUNT = 1u << 3,
+	/**
+	 * FU_CONTEXT_FLAG_BITLOCKER_FDE:
+	 *
+	 * Bitlocker style full disk encryption is in use
+	 *
+	 * Since: 2.0.5
+	 **/
+	FU_CONTEXT_FLAG_BITLOCKER_FDE = 1u << 4,
```

Comment:
What about flipping this around? e.g. FU_CONTEXT_FLAG_FDE_BITLOCKER then we can add other variants as postfixes?

Reply:
Makes sense!  I swapped.